src/main/java/com/revo/deployr/client/data/RBoolean.java

```diff
@@ -23,5 +23,5 @@
      * @return boolean value
      * @see RData
      */
-    public boolean getValue();
+    public Boolean getValue();
```

Comment:
Boolean is an Object which can be assigned as NULL
boolean is primitive and can only have TRUE, FALSE

So when we have NULL values sent as part of input vector , output vector should also contain NULL, hence we changed from boolean to Boolean